src/genaisrc/src/testrun.mts

```diff
@@ -25,16 +25,18 @@ const { generator, output } = env
 
 export async function runTests(
     files: PromptPexContext,
-    options?: PromptPexOptions
+    options?: PromptPexOptions,
+    runGroundtruth: boolean = false
```

Comment:
Extend promptpex options instead of adding argument

Options?: PromptPexOptions & { runGroundTruth?: Boolean }